src/preferences/dialog/dlgprefcontrols.cpp

```diff
@@ -70,9 +70,14 @@ DlgPrefControls::DlgPrefControls(QWidget * parent, MixxxMainWindow * mixxx,
     if (!m_pConfig->exists(ConfigKey("[Controls]","PositionDisplay")))
         m_pConfig->set(ConfigKey("[Controls]","PositionDisplay"),ConfigValue(0));
 
-    if (m_pConfig->getValueString(ConfigKey("[Controls]", "PositionDisplay")).toInt() == 1) {
+    int positionDisplayType = m_pConfig->getValueString(
+            ConfigKey("[Controls]", "PositionDisplay")).toInt();
+    if (positionDisplayType == 1) {
         radioButtonRemaining->setChecked(true);
         m_pControlTrackTimeDisplay->set(1.0);
```

Comment:
You cannot convert a enum class to a double without a cast. 
The idea was to define 
constexpr double k... = 1.0 
to get around this. 

Everything that replaces the 1.0 by a readable text works for me here, I do not care much about type safety here. 
You already use a private enum class inside the widget with the same values. Maybe you find common solution for both.